src/util/ui-options.js

```diff
@@ -90,7 +90,7 @@ export default {
       return this.enableMatterFeatures
     },
     enableServerOnly() {
-      return this.enableMatterFeatures && !this.enableZigbeeFeatures
```

Comment:
What about client clusters which belong to endpoints with Zigbee's zcl device types? Those should still be configurable.

Reply:
yes it appears there is a bigger problem here

Reply:
it does not disable it

Reply:
to be clear, ZAP master branch, as of now, when running CMP mode never distinguishes between Zigbee and Matter. That logic is broken. So this PR fixes that.